src/lib/indexer/__tests__/index.test.ts

```diff
@@ -293,12 +297,52 @@ describe('Indexer', function () {
       const node = withHappyChainNode()
       const handleBlock = sinon.stub().resolves({})
 
-      indexer = new Indexer({ db, node, logger, handleBlock })
+      indexer = new Indexer({ db, node, logger, handleBlock, startupTime, env })
       await indexer.start()
       const result = await indexer.processAllBlocks('1-hash')
 
       expect(result).to.equal(null)
       expect(handleBlock.called).to.equal(false)
     })
   })
+  describe('getStatus tests', function () {
```

Comment:
missing two tests for if we have an activity and that activity was more than 30s ago

Reply:
I'm just counting `if` and `else` combinations. Even if I discount the redundent `if (latestActivityTime === null) throw new Error('sth')` line you have three `if` blocks that return and then the final return. There should be at least 4 tests threrefore (and as you're asserting message actually 5)